lib/beaker-hostgenerator/util.rb

```diff
@@ -1,64 +0,0 @@
-require 'beaker-hostgenerator/data'
-require 'beaker-hostgenerator/data/vmpooler'
-require 'deep_merge'
-
-module BeakerHostGenerator
-  module Utils
-    include BeakerHostGenerator::Data
-    include BeakerHostGenerator::Data::Vmpooler
-
-    def pe_dir(version, family)
-      # If our version is the same as our family, we're installing a
-      # released version. Use the archive path. Otherwise, we want to use
-      # the development build path.
-      if version && family
-        if version == family
-          return "http://enterprise.delivery.puppetlabs.net/archives/releases/#{family}/"
-        else
-          return "http://enterprise.delivery.puppetlabs.net/#{family}/ci-ready"
-        end
-      end
-    end
-
-    def fixup_node(cfg)
-      # PE 2.8 doesn't exist for EL 4. We use 2.0 instead.
-      if cfg['platform'] =~ /el-4/ and pe_version =~ /2\.8/
-        cfg['pe_ver'] = "2.0.3"
-      end
-    end
-
-    def dump_hosts(hosts, path)
```

Comment:
The changes in this file make this backwards-incompatible, which we agreed not to do (no major version bump for this work). Also, there is at least once user of this method:
https://github.com/puppetlabs/pe-rbac-service/blob/master/integration/helpers/installation_helpers.rb#L22